client/src/component/signin/index.jsx

```diff
@@ -0,0 +1,128 @@
+import { React, useState } from 'react';
+import {
+  Layout,
+  Row,
+  Col,
+  Image,
+  Form,
+  Button,
+  Input,
+  message,
+} from 'antd';
+
+import axios from 'axios';
+import 'antd/dist/antd.css';
+import './index.css';
+
+const {
+  Content,
+} = Layout;
+
+function Signin() {
+  const [userData, setUserData] = useState({ email: '', password: '' });
+
+  const handleInputChange = (event) => {
+    const { name, value } = event.target;
+    setUserData({
+      ...userData,
+      [name]: value,
+    });
+  };
+
+  const handleSubmit = async () => {
+    try {
+      // Post data here
+      await axios.post('/api/signin', userData);
+      message.success('Welcome back!');
+    } catch (e) {
+      message.warning('Try again');
+    }
+  };
+
+  return (
+    <Layout>
+      {/* --- Navbar --- */}
+      <Content>
+        <Row
+          justify="end"
+        >
+          <Col span={12} className="login_image">
+            <Image
+              style={{ height: '100vh', width: '100vh' }}
+              preview={false}
+              src="https://i.postimg.cc/9FMJSScj/login-image.png"
+            />
+            <h1 className="text_image">
+              Subscribe with us to make yourself a contributor to charity and help people in need.
+            </h1>
```

Comment:
There is a component in antd called typography, try to use it instead of all your h and p tages